### PR TITLE
chore(Input): migrate story snapshots with effects on cypress side

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2838]

### Description

Migrate Input stories with effects on cypress side

### How to test

- code review, green CI and approving Happo would be enough

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- [x] Ensure that all PR checks are green
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2838]: https://toptal-core.atlassian.net/browse/FX-2838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cypress/component/Input.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+const component = 'Input'
+
+describe('Input', () => {
+  it("renders Input's hovered and focused states", () => {
+    cy.mount(<TestInput />)
+
+    cy.getByTestId('default-input')
+      .as('default-input')
+      .hoverAndTakeHappoScreenshot({
```
Comment (TomasSlama) [inline cypress/component/Input.spec.tsx:25]: ❤️

**Review by TomasSlama — COMMENTED** [review]
It seems to me, that the error state has the same style in the default and hover state, is it expected? 🤔

**Comment by ozgurkececioglu** [comment]
> It seems to me, that the error state has the same style in the default and hover state, is it expected? 🤔

Yeah, I remember a ticket related to that https://toptal-core.atlassian.net/browse/FX-2571. I think we can use this happo screenshot while implementing this ticket. It should definitely be different. I agree 👍

**Review by TomasSlama — APPROVED** [review]

File: cypress/component/Input.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        component,
+        variant: 'error-status/after-hovered',
+      })
+
+    cy.get('@error-input').realClick().get('body').happoScreenshot({
+      component,
+      variant: 'error-status/after-focused',
+    })
+  })
```
Comment (dmaklygin) [inline cypress/component/Input.spec.tsx:47]: wouldn't it be too difficult to search for an issue if this test case fails?

**Review by dmaklygin — APPROVED** [review]

File: cypress/component/Input.spec.tsx (hunk 2 above)
Comment (ozgurkececioglu) [reply cypress/component/Input.spec.tsx:47]: Hmm, I was trying to give meaningful names, and the screenshot also would give an idea about what is broken (I hope). Do you have a better idea? 🙇‍♂️

**Review by OndrejTuma — APPROVED** [review]
Good job 💪

**Pull request merged**